packages/page/src/Application/Message/CopyPageMessage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Page\Application\Message;
+
+class CopyPageMessage
+{
+    /**
+     * @param array{
+     *     uuid?: string
+     * } $identifier
+     * @param array{
+     *     uuid?: string
+     * } $destinationIdentifier
+     */
+    public function __construct(private array $identifier, private array $destinationIdentifier)
```

Comment:
Not sure here kind of miss the `parent` word here.

```suggestion
    public function __construct(private array $identifier, private array $destinationParentIdentifier)
```